### DISCUSSION TITLE
ENV -  Escape the $ sign

### DISCUSSION THREAD

**Discussion by varun-nambiar** [adonisjs/core#2858] [Help] [answered]
According to the documentation - https://docs.adonisjs.com/guides/environment-variables#escape-the--sign   

> **.env**

```
PASSWORD=pa\$\$word
```

returns the expected output  
> **Config.get('app.testing')**
```
pa$$word
```

Whereas  trying the following.
> **.env**
```
PASSWORD=pa\$word
```

returns the expected output  
> **Config.get('app.testing')**
```
pa$$word
```

returns the following
```
pa$
````
Expected output is `pa$word`

**Comment by thetutlage** [comment, marked as answer]
Created an issue here https://github.com/adonisjs/env/issues/25
